exporter/sumologicexporter/README.md

```diff
@@ -10,11 +10,16 @@ Empty string means no compression
 - `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default `1_048_576` (1MB) is used.
 - `metadata_attributes` (optional): List of regexes for attributes which should be send as metadata
 - `log_format` (optional) (logs only): Format to use when sending logs to Sumo. (default `json`) (possible values: `json`, `text`)
-- `metric_format` (optional) (metrics only): Format of the metrics to be sent (default is `prometheus`) (possible values: `carbon2`, `prometheus`)
-  `carbon2` and `graphite` are going to be supported soon.
+- `metric_format` (optional) (metrics only): Format of the metrics to be sent (default is `prometheus`) (possible values: `carbon2`, `graphite`, `prometheus`).
+- `graphite_template` (default=`%{_metric_}`) (optional) (metrics only): Template for Graphite format.
+Placeholders `%{attr_name}` will be replaced with attribute value for `attr_name`. `%{_metric_}` is going to be replaced with metric name.
+Applied only if `metric_format` is set to `graphite`.
 - `source_category` (optional): Desired source category. Useful if you want to override the source category configured for the source.
+Placeholders `%{attr_name}` will be replaced with attribute value for `attr_name`.
```

Comment:
I think this is not entirely clear. Perhaps it should be marked as `graphite-only` or so?

Reply:
you mean `graphite_template`?

Reply:
`source_category` is independent of metric type

Reply:
Oh, right, then I would rephrase the whole section that refers to it so it's clear what's going on. Perhaps something akin to:
> You can specify a template with an attribute for `source_category`, `source_name`, `source_host` or `graphite_template` using `%{attr_name}`. For example, `%{my_attr}`, when there is an attribute `my_attr: my_value` would be expanded to `my_value`. 

And then refer to it for each attribute, like:
> An attribute can be referred using `%{attr_name}`